Bloon/Features/ModTools/BanUser.cs

```diff
@@ -0,0 +1,114 @@
+                if (deleteMessages)
+                {
+                    await discordUser.BanAsync(0, notes);
+                }
+                else
+                {
+                    await discordUser.BanAsync(360, notes);
+                }
+
+                this.LogModAction(ctx.User.Id, discordId, Actions.Banned, notes);
+                await ctx.Message.CreateReactionAsync(DiscordEmoji.FromName(ctx.Client, ":heavy_check_mark:"));
+
+            }
+            catch (DSharpPlus.Exceptions.NotFoundException e)
+            {
+                // User was not found via ID. Tell mod.
+                await ctx.RespondAsync($"Failed to find a guild member with the user ID of `{discordId}`");
+                await ctx.Message.CreateReactionAsync(DiscordEmoji.FromName(ctx.Client, ":warning:"));
+            }
+        }
+
+        [Command("ban")]
+        [Description("Bans a user from SuperBossGames server.")]
+        [Aliases("banhammer", "hammertime")]
+        [ModExclusive]
+#pragma warning disable CS8632 // The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
+        public async Task BanUserAsync(CommandContext ctx, DiscordUser user, bool deleteMessages, [RemainingText] string? notes)
+#pragma warning restore CS8632 // The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
+        {
+            await this.BanUserAsync(ctx, user.Id, deleteMessages, notes);
+        }
+
+        [Command("unbanid")]
+        [Description("Bans a user from SuperBossGames server via ID.")]
+        [ModExclusive]
+#pragma warning disable CS8632 // The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
+        public async Task UnbanUserByIDAsync(CommandContext ctx, ulong discordId, [RemainingText] string? notes)
+#pragma warning restore CS8632 // The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
+        {
+            try
+            {
+                DiscordMember discordUser = await ctx.Guild.GetMemberAsync(discordId);
+
+                await discordUser.UnbanAsync(notes);
+
+                this.LogModAction(ctx.User.Id, discordId, Actions.Unbanned, notes);
+                await ctx.Message.CreateReactionAsync(DiscordEmoji.FromName(ctx.Client, ":heavy_check_mark:"));
+
+            }
+            catch (DSharpPlus.Exceptions.NotFoundException e)
+            {
+                // User was not found via ID. Tell mod.
+                await ctx.RespondAsync($"Failed to find a guild member with the user ID of `{discordId}`");
+                await ctx.Message.CreateReactionAsync(DiscordEmoji.FromName(ctx.Client, ":warning:"));
+            }
+        }
+
+        [Command("unban")]
+        [Description("Unbans a user from the SuperbossGames server.")]
+        [Aliases("unyeet")]
```

Comment:
Why is `unyeet` = unban, but `yeet` = [kick](https://github.com/BloonBot/Bloon/pull/13/commits/9488a9179cbe4a199e177f61dd6524735a733c9a#diff-e71039968905a9ec95f27cdeb6abc8b8cbdcb5537e44f4ee9abafb74ccecd55bR49)?

Reply:
because yes